### ISSUE TITLE
Default Visualisation Colours

### ISSUE THREAD

**Issue by ptheywood** [FLAMEGPU/FLAMEGPU#105]
It doesn't appear that the default visualisation colours, controlled by the `type` value, are documented. 

I.e. Colour `w==1`  maps to `red` etc.

It would be good if this were documented.

Alternatively, some `#DEFINE` or similar could be defined somewhere to simplify the assignment of default visualisation colours.

```
if (lookup.w > 6.5)	                								\n"
	"		colour = vec4(1.0, 1.0, 1.0, 0.0);								    \n"
    "	else if (lookup.w > 5.5)	                								\n"
	"		colour = vec4(1.0, 0.0, 1.0, 0.0);								    \n"
	"	else if (lookup.w > 4.5)	                								\n"
	"		colour = vec4(0.0, 1.0, 1.0, 0.0);								    \n"
    "	else if (lookup.w > 3.5)	                								\n"
	"		colour = vec4(1.0, 1.0, 0.0, 0.0);								    \n"
	"	else if (lookup.w > 2.5)	                								\n"
	"		colour = vec4(0.0, 0.0, 1.0, 0.0);								    \n"
	"	else if (lookup.w > 1.5)	                								\n"
	"		colour = vec4(0.0, 1.0, 0.0, 0.0);								    \n"
    "	else if (lookup.w > 0.5)	                								\n"
	"		colour = vec4(1.0, 0.0, 0.0, 0.0);								    \n"
    "	else                      	                								\n"
	"		colour = vec4(0.0, 0.0, 0.0, 0.0);								    \n"
```

**Comment by Robadob** [comment]
```
7==white
6==magenta
5==cyan
4==yellow
3==blue
2==green
1==red
0==black
```

Should be trivial to expand this to include brown as @mozhgan-kch was after.

**Comment by mozhgan-kch** [comment]
3==navy

**Comment by Robadob** [comment]
`#0000ff` is blue, by definition it's quite literally 100% blue, 0% red, 0% green

**Comment by ptheywood** [comment]
A series of `#define`s would be a sane approach to this, somewhere that is accessible to `functions.c`.

Alternatively an array of fixed size could be used, and the shader generated at (`nvcc`) compile time, using % to wrap things.

**Comment by ptheywood** [comment]
Branch `rc1.5/default-visualisation-colors` (04051ce) adds:
+ Brown colour for values above `7.5`
+ Defines for each colour in `header.h`
    + `agent->state = FLAME_GPU_VISUALISATION_COLOUR_BROWN`;

![image](https://user-images.githubusercontent.com/628937/29577125-3bf5c84c-8762-11e7-982f-dd64e445792e.png)

**Comment by ptheywood** [comment]
This will be merged with #138.
